packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -223,7 +226,13 @@ async function onDelete() {
     contracts.value = contracts.value!.filter(c => !selectedContracts.value.includes(c));
     selectedContracts.value = [];
   } catch (e) {
-    layout.value.setStatus("failed", normalizeError(e, `Failed to delete some of the selected contracts.`));
+    if ((e as Error).message.includes("Inability to pay some fees")) {
```

Comment:
i think if we changed or improved this message from the client we'll have a big here also no?

Reply:
also, this case is handled in the else, the error whatever it will be will appear to the user normally, if you have any idea how to make it better please tell me